src/ensembl/.eslintrc.js

```diff
@@ -29,9 +30,12 @@ module.exports = {
     'react/no-unescaped-entities': 0,
     'react-hooks/rules-of-hooks': 2,
     'prettier/prettier': 0,
-    "no-unused-vars": "off",
+    'no-unused-vars': 'off',
     'no-unneeded-ternary': 'error',
-    'eqeqeq': 'error'
+    'eqeqeq': 'error',
+    'jest/no-focused-tests': 'error',
+    'jest/no-identical-title': 'error',
+    'jest/valid-expect': 'error'
```

Comment:
Yep, I made that error 👍 

![image](https://user-images.githubusercontent.com/6834224/119746646-d0a16580-be88-11eb-8f9f-b82e4dd45fad.png)